scalafmt-core/shared/src/main/scala/org/scalafmt/config/ScalafmtRunner.scala

```diff
@@ -93,11 +93,11 @@ object ScalafmtRunner {
   implicit val encoder: ConfEncoder[ScalafmtRunner] = generic.deriveEncoder
 
   private def overrideDialect[T: ClassTag](d: Dialect, k: String, v: T) = {
-    import org.scalafmt.config.ReflectOps._
     val methodName =
       if (k.isEmpty || k.startsWith("with")) k
       else "with" + Character.toUpperCase(k.head) + k.tail
-    d.invokeAs[Dialect](methodName, v.asParam)
+    val map: Map[String, Any => Dialect] = DialectMacro.dialectMap(d)
+    map(methodName)(v)
```

Comment:
why not `DialectMacro.dialectMap(d)(methodName)(v)` (possibly adding explicit `.apply` if necessary)?